code/game/mecha/mecha.dm

```diff
@@ -35,6 +35,8 @@
 	var/lights = FALSE
 	var/lights_power = 6
 	var/last_user_hud = 1 // used to show/hide the mecha hud while preserving previous preference
+	var/breach_time = 0

```

Comment:
why are these vars not just part of the nevogre anyways? why does breach time even need to be a var?

Reply:
breach time is the time before detonation after destruction, i believe.

Reply:
I know. It is only used in one place on one mech, doesnt need to be a var of all mechs, or even a var at all